corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -189,7 +189,15 @@ <h3 data-bind="text: $root.activeCaseType()"></h3>
                 <i class="sortable-handle fa fa-arrows-v"></i>
               {% endif %}
             </td>
-            <td data-bind="text: name"></td>
+            <td class="main-form">
+                <span data-bind="text: name"></span>
+
+                <input class="form-control"
+                      data-bind="value: $data.label, rows: 1"
+                      placeholder='{% trans "Click here to add a label" %}'
+                      style="margin-top: 8px;"
+                ></input>
+            </td>
```

Comment:
Nice :+1:

Reply:
This needs the screenshots to be updated in the description.

Reply:
added the screenshot for updated view.

Reply:
Is that most recent? It says "click to add a label" but it should have the same content as case property?
Not sure how this looks now, but you could also see how its done when we add a new question to a form in app builder. The text is hazy which looks nice that it is default and can be changed. That would be a nice cosmetic change, so not essential but good from user experience.

Reply:
Yes, that is a recent screenshot, the case_properties were already added, that is why the placeholder is "click to add a label", if a new case_property is added the label is set to the property name.

Reply:
Offlined with Pawan that we would reconsider this difference in behaviour for existing and new case properties.